frontend/src/pages/GamePage.tsx

```diff
@@ -11,16 +12,95 @@ export default function GamePage(){
 
     return(
         <div className={'gamePage'}>
+
+
+
+
+
             <div>
-                <ColourChoice />
+                <fieldset className={'choiceBox'}>
+                    <legend>Farbauswahl</legend>
+                    <ColourChoice />
+                </fieldset>
             </div>
+
             <div>
-                <SolutionColours />
+                <fieldset className={'boxes'}>
+                    <legend>Lösung</legend>
+                    <SolutionColours />
+                </fieldset>
+
+
+                <fieldset className={'guessBox'}>
+                    <legend>12. Versuch</legend>
```

Comment:
Gibt es immer genau 12. Versuche? Wenn ich das richtig sehe, könnte das Ganze um `<Guesses />` mit in die Komponente packen und dann generischer machen, in dem du mit `.map((ele, index) => <Guesses index={index} />)` über die die Menge an Guesses rüber gehst. Dann mit `<legend>{props.index}. Versuch</legend>`. Wäre nur etwas weniger Code, und vor allem wenn du die Anzahl der Versuche Parametrisieren wollen würdest, wäre das perfekt - aber dafür kenne ich das Spiel zu wenig.